sympy/stats/tests/test_continuous_rv.py

```diff
@@ -900,6 +900,33 @@ def test_pareto_numeric():
     # Skewness tests too slow. Try shortcutting function?
 
 
+def test_PowerFunction():
+    alpha = Symbol("alpha", nonpositive=True)
+    a, b = symbols('a, b', real=True)
+    raises (ValueError, lambda: PowerFunction('x', alpha, a, b))
+
+    a, b = symbols('a, b', real=False)
+    raises (ValueError, lambda: PowerFunction('x', alpha, a, b))
+
+    alpha = Symbol("alpha", positive=True)
+    a, b = symbols('a, b', real=True)
+    raises (ValueError, lambda: PowerFunction('x', alpha, 5, 2))
+
+    X = PowerFunction('X', 2, a, b)
+    assert density(X)(z) == (-2*a + 2*z)/(-a + b)**2
+    assert cdf(X)(z) == Piecewise((a**2/(a**2 - 2*a*b + b**2) -
+        2*a*z/(a**2 - 2*a*b + b**2) + z**2/(a**2 - 2*a*b + b**2), a <= z), (0, True))
+    assert simplify(variance(X)) == a**2/18 - a*b/9 + b**2/18
+    assert simplify(E(X)) == a/3 + 2*b/3
```

Comment:
Using simplify in tests isn't recommended. It may slow down the build on travis CI.

Reply:
Okay, I will remove them.